modules/catalog-compaction/src/integrationTest/java/org/apache/ignite/internal/catalog/compaction/ItCatalogCompactionTest.java

```diff
@@ -243,6 +243,27 @@ public void testCompactionRun() {
         expectEarliestCatalogVersion(catalogVersion3 - 1);
     }
 
+    @Test
+    void droppedZoneCompacted() {
+        String ZONE_NAME = "zone1";
+        String TABLE_NAME = "table1";
```

Comment:
variable naming violates checkstyle
```suggestion
        String zoneName = "zone1";
        String tableName = "table1";
```